cloudwatch/src/lambda_function.py

```diff
@@ -77,34 +71,52 @@ def _parse_cloudwatch_log(log, aws_logs_data, log_type):
         pass
 
 
-def _enrich_logs_data(aws_logs_data, context):
-    # type: (dict, 'LambdaContext') -> None
+def _get_additional_logs_data(aws_logs_data, context):
+    # type: (dict, 'LambdaContext') -> dict
+    additional_fields = ['logGroup', 'logStream', 'messageType', 'owner']
+    additional_data = dict((key, aws_logs_data[key]) for key in additional_fields)
+    try:
+        additional_data['function_version'] = context.function_version
+        additional_data['invoked_function_arn'] = context.invoked_function_arn
+    except KeyError:
+        pass
+
     try:
-        aws_logs_data['function_version'] = context.function_version
-        aws_logs_data['invoked_function_arn'] = context.invoked_function_arn
+        # If ENRICH has value, add the properties
+        if os.environ['ENRICH']:
+            properties_to_enrich = os.environ['ENRICH'].split(";")
+            for property_to_enrich in properties_to_enrich:
+                property_key_value = property_to_enrich.split("=")
+                additional_data[property_key_value[0]] = property_key_value[1]
```

Comment:
0 and 1 indices are not that clear - perhaps use constants that clarify that one is the key index and one is the value index?